packages/uui-button/lib/uui-button.element.ts

```diff
@@ -415,6 +415,15 @@ export class UUIButtonElement extends FormControlMixin(
   @property({ type: String })
   public href?: string;
 
+  /**
+   * Set a popovertarget.
```

Comment:
Here is would be good to write the description so it helps the developer understand what target means.

Is it an element id or is it a string like 'bottom'. Also is this where the popover wilbe created within or placed next to ? :-)

Reply:
The description has been changed to:
Set a popovertarget - The id of the popover element which should be triggered on click.